test-integration/test-tools/src/integration_test_context.rs

```diff
@@ -667,6 +667,7 @@ impl IntegrationTestContext {
         ));
     }
 
+    #[allow(clippy::result_large_err)]
```

Comment:
Ideally `#![allow(clippy::result_large_err)]` at the test module level so you don't have to repeat this everywhere.